### ISSUE TITLE
IE iframe error on simple app; Firefox, Chrome have no errors...

### ISSUE THREAD

**Issue by Stephen-McDaniel** [rstudio/shiny#954]
This simple app (very bottom) illustrates some significant issues I am seeing with IE and Shiny Server. These issues do not appear with local Shiny via RStudio. Shiny 0.12.0, Shiny Server 1.3.0.403

Using the below app.R, I see this in the IE Console:

SCRIPT5: Access is denied.
File: iframe.html, Line: 1, Column: 1

The message is in IE only, it doesn't appear in other browsers like Chrome or Firefox.

While this example still renders, more complex examples won't due to similar messages in IE (once again, no errors in other browsers.) Here's where it's links to when I click through the error:
<pre>&lt;html&gt;
&lt;head&gt;
  &lt;meta http-equiv="X-UA-Compatible" content="IE=edge" /&gt;
  &lt;meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /&gt;
  &lt;script&gt;
    document.domain = document.domain;
    _sockjs_onload = function(){SockJS.bootstrap_iframe();};
  &lt;/script&gt;
  &lt;script src="//d1fxtkz8shb9d2.cloudfront.net/sockjs-0.3.min.js"&gt;&lt;/script&gt;
&lt;/head&gt;
&lt;body&gt;
  &lt;h2&gt;Don't panic!&lt;/h2&gt;
  &lt;p&gt;This is a SockJS hidden iframe. It's used for cross domain magic.&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;<code>

# Begin app.R
library(shiny)
library(leaflet)

r_colors <- rgb(t(col2rgb(colors()) / 255))
names(r_colors) <- colors()

ui <- fluidPage(
  leafletOutput("mymap"),
  p(),
  actionButton("recalc", "New points")
)

server <- function(input, output, session) {
 
  points <- eventReactive(input$recalc, {
    cbind(rnorm(40) * 2 + 13, rnorm(40) + 48)
  }, ignoreNULL = FALSE)
 
  output$mymap <- renderLeaflet({
    leaflet() %>%
      addProviderTiles("Stamen.TonerLite",
        options = providerTileOptions(noWrap = TRUE)
      ) %>%
      addMarkers(data = points())
  })
}

shinyApp(ui, server)
# End app.R

**Comment by jcheng5** [comment]
What happens with more complex examples? What version of IE?

**Comment by Stephen-McDaniel** [comment]
More complex versions (add shinydashboard and other graphs) will not render most of the content with the same source error in console and in the debugger. 

I am using IE 11.0.9600.17937 on Win 8.1.

**Comment by Stephen-McDaniel** [comment]
Also just tested this on a cloud PC running IE 11.0.9600.18015 (Windows 7 Kernel via Windows Server R2 2008 Data Center)

**Comment by Stephen-McDaniel** [comment]
For good measure (I wanted to see if leaflet related), here's a shinydashboard/ggvis example. Confirm same message / iframe error. (Example lifted from stackoveflow question.)

library(shiny)
library(shinydashboard)
library(ggvis)

ui <- dashboardPage(
  dashboardHeader(title = "Basic dashboard"),
  dashboardSidebar(),
  dashboardBody(
    # Boxes need to be put in a row (or column)
    fluidRow(
      box(
          width = 4,
          title = "fred",
          ggvisOutput("plot1")
      ),

      box(
        width = 4,
        title = "george",
        ggvisOutput("plot2")
      ),

      box(
        width = 4,
        title = "harold",
        ggvisOutput("plot3")
      )
    )
  )
)

server <- function(input, output) {
  mtcars %>%
    ggvis(~wt, ~mpg) %>%
    layer_points() %>%
    set_options(width = "auto", height = "auto") %>%
    bind_shiny("plot1")

  mtcars %>%
    ggvis(~cyl, ~mpg) %>%
    layer_points() %>%
    set_options(width = "auto", height = "auto") %>%
    bind_shiny("plot2")

  mtcars %>%
    ggvis(~disp, ~mpg) %>%
    layer_points() %>%
    set_options(width = "auto", height = "auto") %>%
    bind_shiny("plot3")
}

shinyApp(ui, server)

**Comment by jcheng5** [comment]
Likely sockjs/Shiny Server issue... cc @fereshtehRS @trestletech @aronatkins

**Comment by jcheng5** [comment]
How about these, do they work for you?
http://shiny.rstudio.com/gallery/kmeans-example.html
http://shiny.rstudio.com/gallery/superzip-example.html

**Comment by Stephen-McDaniel** [comment]
Yes to k-means. However, in the later version of IE (18015), your information box (server.R, ui.R) prevents viewing the output when on the side (shown below.)

Superzip is mixes.  It is good (no error) on earlier IE but iframe error with same link on later version of IE (18015).

<img width="942" alt="050-sidebar" src="https://cloud.githubusercontent.com/assets/9154079/9829140/76d593b4-58ae-11e5-8d1d-f1f7bcfd652a.PNG">

**Comment by Stephen-McDaniel** [comment]
Hi @fereshtehRS @trestletech & @aronatkins ,

Any update on a solution or workaround for this issue? 

Thanks,
Stephen

**Comment by fereshtehRS** [comment]
Stephen,

I have tried your code on IE 11, but cannot see the error you reported.

I did a search on the error and found many reports of this error on
different versions of IE, pointing to settings on the user's computer. For
example this:

http://answers.microsoft.com/en-us/ie/forum/ie10-windows_7/ie10-script5-access-is-denied/e87bdb30-7f2a-4510-bfa3-a22b995f777b?auth=1

I wonder if you are experiencing similar issue.


Fereshteh

On Mon, Oct 19, 2015 at 10:15 PM, Stephen McDaniel <notifications@github.com
